template/src/screens/demoSlice.ts

```diff
@@ -41,28 +19,11 @@ export const demoSlice = createSlice({
     decrementCounterBy: (state, action: PayloadAction<number>) => {
       state.counter -= action.payload
     },
-    getLatestComicAsync: state => {
-      state.comic = Pending(state.comic)
-    },
-    getLatestComicAsyncSuccess: (state, action: PayloadAction<Comic>) => {
-      state.comic = Success(action.payload)
-    },
-    getLatestComicAsyncFailure: (state, action: PayloadAction<Error['message']>) => {
-      state.comic = Failure(action.payload)
-    },
```

Comment:
This boilerplate of 3 actions for calling request and receiving success/failure was replaced by `useQuery`.